URA-695/reproducibility.py

```
@@ -0,0 +1,181 @@
+                       "fastq_L001_R1":"file-G2X22Jj4qyfqzpybKfYJ15VQ",
+                       "fastq_L002_R1":"file-G2X25Q04qyfz1qf57gjxqx5f",
+                       "fastq_L001_R2":"file-G2X22KQ4qyfgpvyy9GQBkgF9",
+                       "fastq_L002_R2":"file-G2X25Qj4qyfZ77jf5kkYV88v"}
+    
+    # set standard variables
+    standard_path = "/reproducibility"
+    standard_project = "project-Gjz0P404fV6bq4J30qBfPBFf"
+    
+    # run sentieon bwa 10 times and store jobs in a list
+    bwa_mem_job_ids = []
+    for i in range(0,10):
```

Ruff (0.8.0)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)